quipucords/api/messages.py

```diff
@@ -195,3 +195,6 @@
     'the invalid filter key %s.'
 REPORT_INVALID_FILTER_QUERY_PARAM = 'One or more filter keys are not '\
     'attributes of the system fingerprint: %s.'
+REPORTS_MISSING_REQUIRED_VALUE = 'A required value for Reports was missing.'
+REPORTS_NO_CSV = 'A CSV file could not be created.'
```

Comment:
Same comment as above - except this occurs if the `deployments.csv` and `details.csv` are both `None`. Does the report exist/ can we give any more info as to why it cannot be created?